api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java

```diff
@@ -254,51 +258,12 @@ public Encounter getEncounter(Integer encounterId) throws APIException {
 		}
 		
 		List<Encounter> encs = new Vector<Encounter>();
-		for (Patient p : Context.getPatientService().getPatients(null, identifier, null, false)) {
+		for (Patient p : Context.getPatientService().getPatients(identifier, null, null, false)) {
```

Comment:
No it was not. If you look at the DAO, these methods were deprecated in favor of PatientDAO#getPatients(String, Integer, Integer) where the first parameter string can either be a name or identifier.